Replicator/Replicator.cc

```diff
@@ -82,11 +82,32 @@ namespace litecore { namespace repl {
         _connectionState = Connection::kConnecting;
         connection()->start();
         // Now wait for _onConnect or _onClose...
+        
+        // handle the unresolved docs
+        C4Error err = { };
```

Comment:
FYI, it's not necessary to initialize C4Error instances. They're initialized by the called function when an error occurs.

Reply:
since this is an internal logic, and we need to make sure the enumeration didn't cause any error. can we use this?

Reply:
understood now. I was thinking not straight. corrected